.github/workflows/generate_manpage.yml

```
@@ -19,7 +19,35 @@ jobs:
           python-version: 3.x
 
       - name: Install dependencies
-        run: pip install argparse-manpage
+        run: |
+          pip install argparse-manpage
+          pip install six
+          pip install pexpect
+          pip install rpm
+
+      - name: Install python3-rpm package with apt-get
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y python3-rpm
```

```suggestion
      - name: Install python3-rpm and python3-dnf package with apt-get
        run: |
          sudo apt-get update
          sudo apt-get install -y python3-rpm python3-dnf
```

The problem that is happening in the CI right now might be fixed with installing python3-dnf together. This is not ideal, but it was the working solution I could find quickly here.